Project/Source Files/Screens/SearchTab/SearchViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SearchTabViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+
+	required init?(coder aDecoder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+	init(searchController: UISearchController = UISearchController(searchResultsController: nil)) {
+		self.searchController = searchController
+		super.init(nibName: nil, bundle: nil)
+	}
+	private var customView: SearchView {
+		return view as! SearchView
+	}
+	private var searchController: UISearchController
+	private var searchScopeButtonTitles = ["Title", "Genre"]
+	override func loadView() {
+		view = SearchView.instanceFromNib()
+	}
+    override func viewDidLoad() {
+        super.viewDidLoad()
+		setupTableView()
+		setupVC()
```

Comment:
those are 2 tabs. Make sure that in xcode project, you have set that 1 intend is 4 spaces, not 1 tab